pro/src/screens/Bookings/BookingsRecapTable/__specs__/BookingsRecapTable.spec.jsx

```diff
@@ -213,6 +219,7 @@ describe('components | BookingsRecapTable', () => {
         booking_token: 'ZEHBGD',
         booking_status: 'validated',
         booking_is_duo: false,
+        booking_id: '1',
```

Comment:
On utiliserait le pattern factory on aurait pas besoin d'ajouter cette valeur à plus de 1 endroit
Si tu as le temps ça pourrait être cool de repasser sur ce fichier de test pour utiliser une factory :) (qui peut être déclarée dans ce fichier si elle sera pas utilisée ailleurs, mais peut être qu'il y en a déjà une existante)

Reply:
En bonus le fichier de test deviendra bcp plus court

Reply:
j'ai déplacé plusieurs test du fichier en jsx dans le fichier en tsx en utilisant la factory pour les bookings, il y a juste un dernier test dans le jsx dont je ne comprends pas l'utilisation, je te laisse regarder ce que j'ai fais. Pour moi le dernier test dans le fichier BookingRecapTable.jsx `should update currentPage when clicking on next page button` n'est pas pertinent donc si on peut l'enlever on pourrait supprimer le fichier.